### DISCUSSION TITLE
environment_vars - Confused with frigate.yml

### DISCUSSION THREAD

**Discussion by HolgerM1** [blakeblackshear/frigate#4464] [Ask A Question] [answered]
Hi,
I'm trying to set up Frigate an J4125 over Hassios as official integration. I'm stuck on the hwaccel. I read that on this Chip I should set LIVA_DRIVERNAME=i965 in config. I can't get rid of this error message. I don't know where the problem is. I'm really noob in YAMLE.
Could somebody help me out?

Thanks!

![Screenshot 2022-11-21 195333](https://user-images.githubusercontent.com/114694650/203136706-93244b84-bb93-42be-80b3-a769ae1a3d36.png)

![Screenshot 2022-11-21 192255](https://user-images.githubusercontent.com/114694650/203136373-15e4fa0e-6a88-45e9-ad69-a7f736a200f2.png)

**Comment by NickM-27** [comment, marked as answer]
It's yaml so it needs to be done in this format:

```yaml
environment_vars:
  LIBVA_DRIVER_NAME: i965
```

**Reply by HolgerM1** [reply]
Wow, thank's it's been descripted as = in the manual. I even tried out the : missing the spaces. I really have no idea about this language.

Thank's again!

**Reply by NickM-27** [reply]
Unfortunately, depending on how frigate is installed there's many different ways to set an environment variable. 

Glad it's working
